src/Generators/Microsoft.Gen.Metering/DiagDescriptors.cs

```diff
@@ -11,123 +11,121 @@ internal sealed class DiagDescriptors : DiagDescriptorsBase
     private const string Category = "Metering";
 
     public static DiagnosticDescriptor ErrorInvalidMethodName { get; } = Make(
-        id: "R9G050",
+        id: "METGEN000",
         title: Resources.ErrorInvalidMethodNameTitle,
         messageFormat: Resources.ErrorInvalidMethodNameMessage,
         category: Category);
 
     public static DiagnosticDescriptor ErrorInvalidParameterName { get; } = Make(
-        id: "R9G051",
+        id: "METGEN001",
```

Comment:
For all of these new descriptors, we'll eventually need to add docs in docs.microsoft.com for them, as well as aka links.

Reply:
Yep, have an issue tracking adding docs and such.

There is docs/list-of-diagnostics.md in this repo. Is that what you mean?

Reply:
> There is docs/list-of-diagnostics.md in this repo. Is that what you mean?

Not exactly, I meant having a `DiagnosticDescriptors.cs` file which is common and used by all projects that use any diagnostic id, and to have that file be the aggregate of all of the different `DiagDescriptors.cs` files we have across the repo as opposed to having those separate. If we already have a central place (even though it is a markdown file as opposed to a C# file) I think I'm okay with that too so really up to you.